### ISSUE TITLE
% in the href attribute are still encoded to %25

### ISSUE THREAD

**Issue by Welloprim** [HakanL/WkHtmlToPdf-DotNet#109]
Hello, 

When I generate a pdf from HTML and use an anchor tag that contains a %. The href keeps encoding the % as %25. 

For example, when I put the cursor on the link : 

- From HTML :
 <img width="275" alt="image" src="https://github.com/HakanL/WkHtmlToPdf-DotNet/assets/44085589/b56930d3-be91-47db-9399-77ddc9e8aa5a">

- From the pdf generated :
<img width="200" alt="image" src="https://github.com/HakanL/WkHtmlToPdf-DotNet/assets/44085589/76b3e87c-3cba-4b5b-9118-94a5edcef697">

Here my code : 

 using(FileStream fs = new FileStream(tempFile, FileMode.OpenOrCreate)) {
     byte[] info = new UTF8Encoding(true).GetBytes(model.Header);
     fs.Write(info, 0, info.Length);
     var doc = new HtmlToPdfDocument() {
       GlobalSettings = {
           ColorMode = ColorMode.Color,
           Orientation = Orientation.Portrait,
           PaperSize = PaperKind.A4Plus,
           DPI = 300,
           Margins = new MarginSettings() {
             Top = 40, Left = 10, Right = 10
           },
         },
         Objects = {
           new ObjectSettings() {
             PagesCount = true,
               HtmlContent = model.Html,
               WebSettings = {
                 DefaultEncoding = "utf-8"
               },
               FooterSettings = {
                 FontSize = 9,
                 Left = DateTime.Now.ToString("MM-dd-yyyy HH:mm:ss"),
                 Right = "Page [page] of [toPage]",
                 Spacing = 2.812
               },
               HeaderSettings = {
                 HtmlUrl = tempFile,
                 Spacing = 10
               },
           },
         },
     };
     pdf = _converter.Convert(doc);
     return pdf;

When debugging the link remains unchanged (not encoded) then after the step `pdf = _converter.Convert(doc); ` I have no visibility because it is a byte[].
Having no workaround at the moment, can I have some help from you, please?

**Comment by HakanL** [comment]
This project is just a wrapper around the native wkhtmltopdf library. Maybe if you test the library directly using command line to see if the issue is in there.

**Comment by Welloprim** [comment]
Indeed, the problem is present in the native library. Thank you for your reactivity
